docs/api/commands/uncheck.mdx

```diff
@@ -57,11 +57,28 @@ Pass in an options object to change the default behavior of `.uncheck()`.
 | `timeout`                    | [`defaultCommandTimeout`](/guides/references/configuration#Timeouts)           | Time to wait for `.uncheck()` to resolve before [timing out](#Timeouts)                                                                            |
 | `waitForAnimations`          | [`waitForAnimations`](/guides/references/configuration#Actionability)          | Whether to wait for elements to [finish animating](/guides/core-concepts/interacting-with-elements#Animations) before executing the command.       |
 
+### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)
```

Comment:
I don't mind how this reads, but I do not believe the section `Syntax` is the correct heading for these concepts since this is more than the syntax at this time.

Reply:
I haven't committed these changes / applied them to every command yet because it'd involve manually moving around each file for a direction I'm not 100% sure on yet.

Reply:
Applied it to the first three commands: .and(), .as(), and .blur(), so people can see it 'live'. If we like the direction I'll go through and apply it to all commands.